src/redis-cli.c

```diff
@@ -2439,7 +2437,7 @@ clusterManagerCommandDef clusterManagerCommands[] = {
     {"set-timeout", clusterManagerCommandSetTimeout, 2,
      "host:port milliseconds", NULL},
     {"import", clusterManagerCommandImport, 1, "host:port",
-     "from <arg>,copy,replace"},
+     "from <arg>,from-user <arg>,from-pass <arg>,from-askpass,copy,replace"},
```

Comment:
i don't understand... how is it used?
this help doesn't seem to reflect this code: `(!strcmp(argv[i],"--cluster-from-user")`
@hwware can you update the top comment of the PR?

Reply:
Hi @oranagra , The cluster management related help is built in this function https://github.com/redis/redis/blob/unstable/src/redis-cli.c#L6548 .. It will automaticallly append the --cluster- prefix in the help. You can also try "./redis-cli --cluster help" to see that...  thanks!

Reply:
ohh, thanks